config.go

```diff
@@ -54,8 +54,8 @@ func NewDefaultConfig() Config {
 		EventsPort:        10001,
 		EventsInterface:   "eth0",
 		LogOutput:         ioutil.Discard,
-		HttpBasicAuthUser: "",
-		HttpBasicPassword: "",
+		HTTPBasicAuthUser: "",
```

Comment:
Since were touching it ... should we remove the defaulted fields? ...  i.e. HTTPBasicAuthUser, HTTPBasicPassword and CallbackURL? ... all of which would be empty anyhow ...